docs/hub/datasets-usage.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Using Datasets
+---
+
+<h1>Using Datasets</h1>
+
+Once you've found an interesting dataset on the Hugging Face Hub, you can proceed to load the dataset using 🤗 Datasets. Many datasets on the Hub contain a [loading script](https://huggingface.co/docs/datasets/dataset_script), which allows you to easily [load the dataset when you need it](https://huggingface.co/docs/datasets/load_hub).
+
+Some datasets might not include a loading script, in which case the data might be stored directly in the repository. The 🤗 Datasets can be used to [load those kinds of datasets](https://huggingface.co/docs/datasets/loading#hugging-face-hub) as well. For more information about using datasets, there are plenty of [tutorials](https://huggingface.co/docs/datasets/tutorial) and [how-to guides](https://huggingface.co/docs/datasets/how_to) available in the 🤗 Datasets documentation.
```

Comment:
maybe add "for example CSV, JSON and arrow files (full list in the datasets doc)"

Reply:
(nit) "For example CSV, JSON and Parquet files" -> we encourage Parquet over Arrow for cold storage.

Maybe we can have a page in the `datasets` docs to list all the supported data and compression formats, instead of redirecting here to the "how to load data from the Hub" part of the docs.

Reply:
Sure, I can add `for example CSV, JSON and Parquet files`. When a page gets added to `datasets` with the supported formats, I can link to that as well!